tests/io/feed_parsers/nitf_tests.py

```diff
@@ -88,7 +88,7 @@ def test_anpa_category(self):
         self.assertEqual(self.item.get('anpa_category')[0]['qcode'], 'a')
 
     def test_word_count(self):
-        self.assertEqual(349, self.item.get('word_count'))
+        self.assertIsInstance(self.item.get('word_count'), int)
```

Comment:
Checking the type is ok, but we also must ensure the count equality.

Reply:
The question is what is the correct number? It's not practical to go over hundreds of words to count it manually in order to be able to compare the results. There are also various ways to count formatting. Even if you counted words manually it's likely that you would get a different result than it was specified in the assert. That's why I prefer to stick to specific scenarios in `text_utils.py`.

Reply:
In this case what all these test_word_count tests do?

Reply:
ok, let's keep it as it is

Reply:
> In this case what all these test_word_count tests do?

I would remove them, but kept just to check that there's some number populated. Not sure if it's very useful.